my-app/README.md

```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Consider moving the project outside the "my-app" folder.

Reply:
/fxd